doofinder.php

```diff
@@ -338,6 +367,25 @@ protected function isConfigured()
         return Db::getInstance()->getValue($sql);
     }
 
+    protected function isSectorConfigured($shop_id)
+    {
+        $shops = Shop::getShops();
+        usort($shops, function ($a, $b) {
+            return $a['id_shop'] - $b['id_shop'];
+        });
```

Comment:
You can use `getSortedShops()` in this case

Reply:
Thanks, I missed replacing that one.